src/auth/auth.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Controller, Body, Post, HttpCode, UseGuards } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { AuthDto } from './dto';
+import { LocalAuthGuard } from './guards/local-auth.guard';
+import { User } from '@prisma/client';
+import { AuthGuard } from '@nestjs/passport';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @HttpCode(200)
+  @UseGuards(LocalAuthGuard)
+  @Post('signin')
+  async signIn(@Body() user: User) {
+    return await this.authService.signIn(user);
+  }
+
+  @HttpCode(201)
+  @Post('signup')
+  async singUp(@Body() dto: AuthDto) {
```

Comment:
for this method also check the above comment

Reply:
also one more thing - I would use separate request dtos for `signIn` and `signUp` method. On a first glance were introducing redundancy but on the other hand we are gaining decoupling. So here we would have e.g.
```suggestion
  async singUp(@Body() signUpRequest: SignUpRequestDto) {
```

Reply:
also, noticed just now - there is a typo in `singUp` - should be `signUp` :)